src/scenes/PostGig.jsx

```
@@ -287,3 +147,155 @@ export const PostGig = () => {
 }
+
+
+const ReportPhase = ({ financials, onNext }) => (
+  <div className='space-y-6'>
+    <div className='grid grid-cols-2 gap-8'>
+      {/* Income Column */}
+      <div>
+        <h3 className='text-xl border-b border-(--toxic-green) mb-4'>INCOME</h3>
+        <ul className='space-y-2 text-sm'>
+          {financials.income.breakdown.map((item, i) => (
+            <li key={i} className='flex justify-between'>
```

Biome (2.3.14)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)